samples/MAUI/LibVLCSharp.MAUI.Sample.MediaElement/MauiProgram.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace LibVLCSharp.MAUI.Sample.MediaElement
+{
+    /// <summary>
+    /// The MauiProgram class is responsible for creating and configuring the MAUI application.
+    /// </summary>
+    public static class MauiProgram
+    {
+        /// <summary>
+        /// Creates and configures the MAUI application.
+        /// </summary>
+        /// <returns>The configured MAUI application.</returns>
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+                .UseMauiApp<App>()
+                .ConfigureFonts(fonts =>
```

Comment:
Is that still necessary? Seems it's done in `src/LibVLCSharp.MAUI/AppHostBuilderExtensions.cs`